common/src/test/resources/archunit_store/package-cycles.txt

```diff
@@ -0,0 +1,116 @@
+Cycle detected: Slice org.axonframework.common.configuration -> \
```

Comment:
Just so I comprehend, this file will be there until we resolve these issues in a future major release?